src/main/java/net/atos/zac/signalering/event/SignaleringEventObserver.java

```diff
@@ -102,12 +103,17 @@ private Signalering getSignaleringVoorGroup(final SignaleringEvent<?> event, fin
         return null;
     }
 
-    private Signalering getSignaleringVoorBehandelaar(final SignaleringEvent<?> event, final Zaak subject,
-            final ZaakInformatieobject detail) {
+    private @Nullable Signalering getSignaleringVoorBehandelaar(
+            final SignaleringEvent<?> event,
+            final Zaak subject,
+            final ZaakInformatieobject detail
+    ) {
         final Optional<Rol<?>> behandelaar = getRolBehandelaarMedewerker(subject);
         if (behandelaar.isPresent()) {
```

Comment:
Optional mapping is useful here

Reply:
Not for this PR I would say. This PR is already quite big.